enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -790,7 +790,32 @@ To answer this question, we need to understand the requirements around feature
 gates and understand whether just installing the gatewayclasses CRD may be
 problematic for some clusters.
 
-**Resolution**: TBD.
+**Resolution**: Yes, we should use a feature gate.  It is not just the installation of
+the CRDs that must be considered.  The additional controllers and watchers needed for
+managing Gateway API resources should be avoided if not needed. In addition, having an
+official feature gate is useful for setting expectations about whether a cluster may be
+upgraded. In Dev and Tech Preview, a cluster with this feature should not be upgradeable.
+Finally, if a user wants to install the Gateway API CRDs on their cluster for experimenting
+with an implementation other than OSSM/Istio, we should permit them to do this without
+interference from OpenShift Gateway API on OSSM.
+
+In Dev Preview we choose to use the `CustomNoUpgrade` variety of feature gate primarily
+because our other choice was named `TechPreviewNoUpgrade`, and we felt this would cause some
+confusion over the status of the feature.  Even though `TechPreviewNoUpgrade` has automatic
+CI testing in place and is managed by the OpenShift API, we won't need it until the feature has
+graduated in maturity to Tech Preview.
+
+The list of feature gate constraints includes:
+
+* The feature gate should be named `GatewayAPI` for all iterations of the feature.
+* The GatewayAPI feature gate should be a `CustomNoUpgrade` variety in Dev Preview,
+a `TechPreviewNoUpgrade` variety in Tech Preview, and removed before GA.
```

Comment:
I'm not sure whether "variety" is the right terminology, but I guess the terminology for this isn't formally defined (it certainly isn't _clearly_ defined).  Anyway, I thought the feature gate could be kept for the first GA release, but changed to be on by default, in order to provide an off switch in case problems are found when it hits a wider audience.